apps/src/code-studio/components/progress/UnitCalendar.jsx

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import _ from 'lodash';
+import UnitCalendarLessonChunk from './UnitCalendarLessonChunk';
+import {unitCalendarLesson} from '@cdo/apps/templates/progress/unitCalendarLessonShapes';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import color from '@cdo/apps/util/color';
+
+const weekWidth = 592;
```

Comment:
I think the convention for constants might be all caps separated by underscores but I'm not 100% sure on that. I can't find it in the style guide so that might just be a style of coding I picked up from other parts of our code base.